### PR TITLE
Allow for non tfm- packages on Fedora and RHEL8

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
@mmoll any idea whats causing this stdlib failure?

**Comment by mmoll** [comment]
This needs (at least) a new theforeman-{dhcp, dns, tftp} release.

File: manifests/plugin/dynflow.pp [hunk 1]
```diff
@@ -50,14 +50,14 @@
     template_path => 'foreman_proxy/plugin/dynflow.yml.erb',
   }
 
-  if $::osfamily == 'RedHat' and $::operatingsystem != 'Fedora' {
-    $scl_prefix = 'tfm-'
-  } else {
-    $scl_prefix = '' # lint:ignore:empty_string_assignment
-  }
+  if $::osfamily == 'RedHat' {
```
Comment (ekohl) [inline manifests/plugin/dynflow.pp]:
```suggestion
  if $facts['osfamily'] == 'RedHat' {
```

**Comment by ehelms** [comment]
@ekohl Added your suggested commit

**Comment by ekohl** [comment]
@iNecas this does mean we don't run smart_proxy_dynflow_core on EL8 (like Debian-based distros). That's correct, right?

**Comment by iNecas** [comment]
Not exactly, to keep the behavior between rhel7 and rhel8 consistence, I believe we want to update https://github.com/theforeman/smart_proxy_dynflow/blob/51bfa8089ce97ee47003f63eafe20581959c5356/lib/smart_proxy_dynflow/plugin.rb#L8 to enforce use of separate smart_proxy_dynflow_core service even when the gem is actually available.

**Comment by ekohl** [comment]
Right, so that's sort of what I was hinting at. This current code runs the service but it's not actually used because of that code. This probably needs a proper redmine issue.

**Comment by iNecas** [comment]
Here we go https://projects.theforeman.org/issues/26969

**Pull request merged**